hugo/layouts/partials/langium-scripts.html

```diff
@@ -1,4 +1,4 @@
-<script>
+{{/*  <script>
     // try {
```

Comment:
Commented-out code should be removed... it is an indicator that the code is not finished yet. Normally when it is production ready you should be free of it.